content/media-player/_index.md

```diff
@@ -0,0 +1,73 @@
++++
+title = "Advanced Analytics for Media Player"
+menuTitle = "Introduction"
+pre = "<i class='fas fa-rocket'></i> "
+chapter = false
+weight = 1
++++
+
+Welcome to the **Advanced Analytics for Media Player** accelerator. Once finished, you will be able to build a deeper understanding of user behavior across your media players and use your data to influence business decisions.
```

Comment:
Just curious, why are we calling this Advanced Analytics for Media Player accelerator? Guessing it is a marketing decision, it seems a bit complicated.

Reply:
John suggested "Video and Media Analytics" which I think is a bit clearer. Unless no other ideas come up, I'll swap it to this